Workbooks/Workloads/SQL Insights/SQL Insights.workbook

```diff
@@ -206,20 +206,6 @@
             "resourceType": "microsoft.resourcegraph/resources",
             "label": "Total computer count"
           },
-          {
-            "id": "763c4b4e-d6f0-4666-b19f-6caf14a534fd",
-            "version": "KqlParameterItem/1.0",
-            "name": "SqlServerMapping",
-            "type": 1,
-            "query": "Resources\n| where type =~ 'microsoft.sql/servers'\n| summarize Data = tostring(makelist(pack('id', id, 'name', name), 10000))",
-            "crossComponentResources": [
-              "value::all"

```

Comment:
Should this be checking all subscriptions, or just the ones in scope?

Reply:
I just removed this code. Should have been All in previous case as the targeted SQL Azure Dbs could be in a different sub than the VM.